.github/pr-custom-review.yml

```diff
@@ -0,0 +1,8 @@
+rules:
+  - name: FILES
+    condition: .*/runtime/.*lib.rs
+    check_type: changed_files
+    min_approvals: 2
+    users:
```

Comment:
`users:` can be deleted because it specifies an empty group, which is meaningless

Reply:
Removed unused section